kpet/__init__.py

```diff
@@ -44,6 +44,8 @@ def main(args=None):
     common_parser = argparse.ArgumentParser(add_help=False)
     common_parser.add_argument('--verbose', '-v', action='count')
     parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('-d', '--debug', action='store_true',
+                        help='Enables debug (backtrace) output on error')
```

Comment:
Could you put it in imperative mood? I.e. write "Enable", not "Enables"?

Reply:
Absolutely, sorry I keep doing that